src/idl_parser.cpp

```diff
@@ -276,9 +275,7 @@ static void SimpleQsort(T *begin, T *end, size_t width, F comparator, S swapper)
   SimpleQsort(r, end, width, comparator, swapper);
 }
 
-
-template<typename T>
-static inline void SingleValueRepack(Value &e, T val) {
+template<typename T> static inline void SingleValueRepack(Value &e, T val) {
```

Comment:
Lots of these are just format change. Maybe move those to a separate PR first to avoid noise?